mithril-aggregator/src/http_server/routes/artifact_routes/cardano_transaction.rs

```diff
@@ -120,7 +123,7 @@ pub mod tests {
     }
 
     #[tokio::test]
-    async fn test_cardano_transactions_get_ok() {
+    async fn test_cardano_transactions_get_ok() -> Result<(), String> {
```

Comment:
Maybe you should use an `unwrap` or `expect` in the body of the test function instead of returning a `Result` (as we usually do in all the tests)?